core/src/test/java/com/linecorp/armeria/server/auth/AuthorizerTest.java

```diff
@@ -126,17 +129,71 @@ public void orElseSecond() {
 
     @Test
     public void orElseToString() {
-        final Authorizer<Object> a = new AuthorizerWithToString("A");
-        final Authorizer<Object> b = new AuthorizerWithToString("B");
-        final Authorizer<Object> c = new AuthorizerWithToString("C");
-        final Authorizer<Object> d = new AuthorizerWithToString("D");
+        final Authorizer<Object> a = new NamedAuthorizer<>("A");
+        final Authorizer<Object> b = new NamedAuthorizer<>("B");
+        final Authorizer<Object> c = new NamedAuthorizer<>("C");
+        final Authorizer<Object> d = new NamedAuthorizer<>("D");
+        final Authorizer<Object> e = new NamedAuthorizer<>("E");
 
         // A + B
         assertThat(a.orElse(b).toString()).isEqualTo("[A, B]");
-        // A + B
+        // A + B + C
         assertThat(a.orElse(b).orElse(c).toString()).isEqualTo("[A, B, C]");
-        // (A + B) + (C + D)
-        assertThat(a.orElse(b).orElse(c.orElse(d)).toString()).isEqualTo("[A, B, C, D]");
+        // A + B + (C + D) + E
+        assertThat(a.orElse(b).orElse(c.orElse(d)).orElse(e).toString()).isEqualTo("[A, B, [C, D], E]");
+    }
+
+    @Test
+    public void orElseHandler() {
```

Comment:
Done. Used `@MethodSource` instead of `@CsvSource` since JUnit 5 lacks array parameter conversions. Also migrated the unit test fully to JUnit 5 as `@ParameterizedTest` was in conflict with Junit 4.

Reply:
Thank you for migration, @max904-github :bow: